src/test/java/teammates/sqlui/webapi/BinFeedbackSessionActionTest.java

```diff
@@ -0,0 +1,183 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import org.mockito.Mockito;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.storage.sqlentity.Instructor;
+import teammates.ui.webapi.BinFeedbackSessionAction;
+
+/**
+ * SUT: {@link BinFeedbackSessionAction}.
+ */
+public class BinFeedbackSessionActionTest extends BaseActionTest<BinFeedbackSessionAction> {
+
+    private Course typicalCourse = getTypicalCourse();
```

Comment:
since this variable is only used in the `setUpMethod()` method, let's minimize the scope and only define it there